Scripts/summaryStatistics/biophysics/anisotropy.py

```diff
@@ -34,7 +34,7 @@ def get_order_parameter(cell_angles):
     """
     Calculates the order parameter for a list of cells
```

Comment:
According to what Brian and I have discussed, the anisotropy that is first written by Jonathon and then edited by me is fine and correct. So, why do you use this? you can find the method here: https://pubmed.ncbi.nlm.nih.gov/33378880/

Reply:
Duplicated code? Let's compare to confirm we've got the best analysis

Reply:
Since we are analyzing anisotropy over an microcolony, I believe it makes sense to have a calculation for a global order parameter. The function calculates a scalar order parameter for the entire colony (i.e. global order parameter). Click [here](https://arxiv.org/pdf/1409.3542.pdf) for some background. Presumably, Jonathan's [code](https://github.com/ingallslab/bsim-related/blob/main/bsim_related/data_processing/cell_data_processing.py) will return the same result if the neighbour search-radius is infinitely large.

Other details: the method in my commit is O(n) in complexity. Searching through all cells for each cell and then taking the average of local order parameters is O(n^2).

@bpingalls I believe we should keep both methods around; orientation of micropatches within a colony may be of interest.